projects/novo-elements/src/elements/form/extras/file/extras/file/File.spec.ts

```diff
@@ -23,10 +23,20 @@ describe('Class: NovoFile', () => {
   });
 
   describe('Method: read()', () => {
-    it('should read files.', () => {
-      jest.spyOn(file.reader, 'readAsDataURL').mockImplementation(() => {});
-      file.read();
-      expect(file.reader.readAsDataURL).toHaveBeenCalled();
+    it('should read files.', async () => {
+      return new Promise((resolve) => {
+        jest.spyOn(file.reader, 'readAsDataURL').mockImplementation(() => {});
+        file.read().then(() => {
+          expect(file.fileContents).toEqual('The contents');
+          resolve(true);
+        });
```

Comment:
Why using `then()` in a `async` function and not `await`?

Reply:
It will never actually resolve to true.  I need to explicitly call the onload to resolve the promise